src/Resources/translations/ResetPasswordBundle.ca.xlf

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<xliff xmlns="urn:oasis:names:tc:xliff:document:1.2" version="1.2">
+    <file original="file.ext" source-language="en" datatype="plaintext">
```

Comment:
Missing `target-language` attr, see any diffrent from `en` translation:

```suggestion
    <file original="file.ext" source-language="en" target-language="ca" datatype="plaintext">
```

Reply:
I've added the target language attribute, nevertheless a lot of other language files hasn't this attribute.

Reply:
May be not "a lot" but "some" ;-)